tools/generate-github-jwt.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
```

Comment:
Were you just using this file locally? Or is it used somewhere in the workflows?